future/problem.md

```diff
@@ -2,14 +2,66 @@
 title: Defining the problem
 ---
 
-We focus on solving the right problems, the right way. In other words, once we understand the problem, we are comfortable pivoting to different solutions, not to different markets. It's critical that you deeply understand the problem you're solving. Just as importantly, you need to bring everyone along with you. Your partners and teammates need to understand, and agree on, the problem you're tackling.
+It's critical that you deeply understand the problem you're solving. This means that once you understand the problem, you are comfortable exploring and pivoting to different solutions. 
 
-Below are some outcomes 18F product managers drive when defining the problem space:
+Just as importantly, you need to bring everyone along with you. Your partners and teammates need to agree on the problem you're tackling and the problems you are not tackling. A well-written problem statement helps align and motivate your team around a common goal, providing focus and keeping distractions at bay.
 
-- Surface the major challenges and define the overlying problem
-- Identify the audiences affected by the problem
-- Understand and articulate the impact of the problem (who is affected and by how much)
-- Validate the problem with user and/or market research
-- Get key stakeholders on board and committed to solving the problem
-- Document the problem throughout implementation and keep it visible and central. Refer to it regularly
-- Regularly reconsider the problem and update as appropriate
+###Considerations: A good problem statement
```

Comment:
let's talk with @c-project-resources. I know we have a lot of different problem statement docs roaming around, and we should probably determine if this is the definitive one, or if it should move somewhere else. I don't consider this a blocker, just something we should figure out.